gpkit/feasibility.py

```diff
@@ -0,0 +1,85 @@
+import numpy as np
+
+from .variables import Variable, VectorVariable
+from .substitution import get_constants
+from .varkey import VarKey
+from .posyarray import PosyArray
+
+
+def feasibility_model(model, flavour="max", varname=None, constants=None):
+    """Returns a new GP for the closest feasible point of the current GP.
```

Comment:
Yeah. The docstring was written before coding, and there was a bit of creep...

The trick is, when you make a feasibility model it's nice to know what the newly-created variables are and correspond to. I guess this could be done by returning a GP/Model with extra attributes, instead of this crazy tuple? (I thought about making it a dict, though that's also ungainly).